pages/index.vue

```diff
@@ -23,6 +23,7 @@ privacy,opensource,accounting,asset-management,taxes,tax-reporting`
 
 export default defineComponent({
   name: 'Index',
+  middleware: 'maintenance',
```

Comment:
We don't really need the middleware in index, users should be able to access the landing page even if the whole backend is down.